### DISCUSSION TITLE
Community Showcase

### DISCUSSION THREAD

**Discussion by unamashana** [magicbell/community#3] [General]
We are excited to see to see MagicBell in your product, and hear more about how you are leveraging it. 

As a thank you, we will be sending every contributor a [MagicBell swag Box](https://user-images.githubusercontent.com/5238698/207850609-f747e27d-17cc-4578-9551-11511a25c5fe.jpeg).

**Comment by doron2402** [comment]
Hey!
I'm Doron from [Try Per Diem](https://tryperdiem.com) we're fans of MagicBell!

**Comment by FINDarkside** [comment]
We're using MagicBell to build new platform for Inderes at [beta.inderes.fi](https://beta.inderes.fi), [inderes.se](https://www.inderes.se) and [inderes.dk](https://www.inderes.dk)

Inderes’ mission is to connect listed companies and investors. Our customer base comprises over 150 listed companies that want to serve their current and potential shareholders by providing better investor communications. Our investor community consists of more than 75,000 active members interested in investing in stocks. Our social objective is to democratize information in the financial markets, so that everyone has equal opportunity to invest and save for their own future.

We already had our own systems for push notifications, in-app inbox and email notifications at our (soon legacy) site [inderes.fi](https://www.inderes.fi), but we needed to rebuild it for our new site. Using MagicBell instead of building our own system saved us months of developer time. ![Screenshot from 2022-12-16 17-16-12](https://user-images.githubusercontent.com/8396424/208129584-1e5507d9-d58f-4f7e-9540-07d6b1426c09.png)

Notifications are very important part of our service because we publish so much content that it's hard to keep up with everything unless you can get notifications only about the companies and content you're interested in. With MagicBell users can now even have separate notification delivery settings for different type of content with very little effort from our end. We've used the `custom_attributes` part of MagicBell extensively to achieve exactly what we need. To send notifications to people who have subscribed to companies which the content is about and only to users who understand the language said content is in.
![Screenshot from 2022-12-16 17-31-14](https://user-images.githubusercontent.com/8396424/208132760-5a2108b8-f7a9-481f-9c6b-b3ac2b4d7138.png)

**Reply by unamashana** [reply]
Thank you, @FINDarkside. Also, we appreciate all the bug reports (with detailed reproduction steps) you share with us in the MagicBell Power Users Slack!

**Reply by 3v0k4** [reply]
Wow, thanks @FINDarkside for sharing 🙏

> Our social objective is to democratize information in the financial markets, so that everyone has equal opportunity to invest and save for their own future.

Fantastic!

**Comment by samirrayani** [comment]
We're using Magic Bell to build Beam ([heybeam.com](https://heybeam.com)) ! We were able to get in-app and email notifications working very quickly and are huge fans of the headless libraries

![Screenshot 2022-12-21 at 2 12 23 PM](https://user-images.githubusercontent.com/179782/208984817-bba5e64e-5c17-4376-91cb-0e73a458669b.png)

![Screenshot 2022-12-21 at 2 12 46 PM](https://user-images.githubusercontent.com/179782/208984891-d0f9f8f0-b930-467f-b4d7-de4b6a4e4b02.png)

**Comment by gagan94** [comment]
Hello !
We are using Magicbell in our product at [Mailmodo](https://www.mailmodo.com/) . We at Mailmodo are building the next generation email marketing solution and we frequently send notification to our users regarding the updates of different activities performed by the user, such as Campaign after processing, Account activation, Email credit usage, Integration enablement, Bounce rate indication etc.

 Magicbell has made it easy for our customers to be aware of different things happening  in the background and it also provides an easier way to not bombard the users with multiple notifications as well.


<img width="1274" alt="image" src="https://user-images.githubusercontent.com/261239/209532986-cff61a8a-2d8c-4914-a9e1-1e7355fab98e.png">

**Comment by naregschoolwork** [comment]
Hey everyone,
We are using MagicBell for our notification system for a school project and I really love the simplicity of it. Furthermore, it has an amazing community that gives useful feedback through Slack. :) Here is what our notification system looks like:
![image](https://user-images.githubusercontent.com/67912337/225482176-cc9d3e72-dfab-4b86-8ff4-d83b471da904.png)

**Comment by Marilinas** [comment]
We had no trouble getting in-app and email notifications up and running, and we think the headless libraries are fantastic, [run 3](https://run3online.io)
